docs/weaviate/concepts/vector-quantization.md

```diff
@@ -108,34 +109,63 @@ The size of the training set is configurable. The default is 100,000 objects per
+:::
+
 ## Over-fetching / re-scoring
 
-Weaviate over-fetches results and then re-scores them when you use SQ or BQ. This is because the distance calculation on the compressed vectors is not as accurate as the same calculation on the original vector embedding.
+Weaviate over-fetches results and then re-scores them when you use SQ, RQ, or BQ. This is because the distance calculation on the compressed vectors is not as accurate as the same calculation on the original vector embedding.
 
 When you run a query, Weaviate compares the query limit against a configurable `rescoreLimit` parameter.
 
 The query retrieves compressed objects until the object count reaches whichever limit is greater. Then, Weaviate fetches the original, uncompressed vector embeddings that correspond to the compressed vectors. The uncompressed vectors are used to recalculate the query distance scores.
 
 For example, if a query is made with a limit of 10, and a rescore limit of 200, Weaviate fetches 200 objects. After rescoring, the query returns top 10 objects. This process offsets the loss in search quality (recall) that is caused by compression.
 
-:::tip
-Learn more about how to [configure binary quantization](../configuration/compression/bq-compression.md) in Weaviate.<br/><br/>
-You might be also interested in our blog post [32x Reduced Memory Usage With Binary Quantization](https://weaviate.io/blog/binary-quantization).
+:::note RQ optimization
+With RQ's high native recall of 98-99%, you can often disable rescoring (set `rescoreLimit` to 0) for maximum query performance with minimal impact on search quality.
 :::
 
 ## Vector compression with vector indexing
 
 ### With an HNSW index
 
-An HNSW index can be configured using [PQ](#product-quantization) or [BQ](#binary-quantization). Since HNSW is in memory, compression can reduce your memory footprint or allow you to store more data in the same amount of memory.
+An [HNSW index](./indexing/vector-index.md#hierarchical-navigable-small-world-hnsw-index) can be configured using [PQ](#product-quantization), [SQ](#scalar-quantization), [RQ](#rotational-quantization), or [BQ](#binary-quantization). Since HNSW is in memory, compression can reduce your memory footprint or allow you to store more data in the same amount of memory.
 
 :::tip
 You might be also interested in our blog post [HNSW+PQ - Exploring ANN algorithms Part 2.1](https://weaviate.io/blog/ann-algorithms-hnsw-pq).
 :::
 
 ### With a flat index
 
-[BQ](#binary-quantization) can use a flat index. A flat index search reads from disk, compression reduces the amount of data Weaviate has to read so searches are faster.
+[BQ](#binary-quantization) can use a [flat index](./indexing/inverted-index.md). A flat index search reads from disk, compression reduces the amount of data Weaviate has to read so searches are faster.
+
+:::note
+RQ is not currently supported with flat indexes.
```

Comment:
This seems redundant given the above. Wdyt? (Otherwise it's a bit odd that we are silent on PQ / SQ)